source/reference/config/triggers.txt

```diff
@@ -61,6 +72,11 @@ trigger type. The following fields exist in *all* trigger configuration files:
        - ``"AUTHENTICATION"``
        - ``"SCHEDULED"``
 
+   * - | ``disabled``
+       | ``boolean``
+     - If ``true``, the trigger will not listen for any events and will
```

Comment:
i assume this is enabled by default? (default is `false`)

Reply:
In the UI, this is true. In the CLI, you can set it as true or false in the config. Will leave out.